webpack/assets/javascripts/react_app/components/toast/toastListitem/index.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Icon from '../../common/Icon';
+import helpers from '../../../common/helpers';
+import CloseButton from '../../common/CloseButton';
+import Timer from '../../../common/Timer';
+import { defaultTimerDelay } from './Toast.consts';
+
+class Toast extends React.Component {
+  constructor(props) {
+    super(props);
+    helpers.bindMethods(this, [
+      'onMouseEnter',
+      'onMouseLeave'
+    ]);
+  }
+
+  componentDidMount() {
+    const { sticky, dismiss, timerDelay } = this.props;
+
+    if (!sticky) {
```

Comment:
how would you do that? an action is an action and a time is a timer.
at first i was eager to make changes to the timer also, but the more i thought about it, the more it makes sense... and it can be reused in other places in the app...